src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImportDialog.java

```diff
@@ -25,12 +26,13 @@
    private DataImport dataImport_;
    
    public DataImportDialog(String caption,
+                           SourceServerOperations server,
```

Comment:
Once you get a hold of the server in DataImport via injection you'll be able to get rid of it from here.